### DISCUSSION TITLE
Probably this needs immadiate attention

### DISCUSSION THREAD

**Discussion by ghost** [phcode-dev/phoenix#1764] [General]
I could be able to reach this server system with following this menu. Seems dangerous to me. 

<img width="1929" alt="Screenshot 2024-07-20 at 01 34 55" src="https://github.com/user-attachments/assets/7eb5f138-f6e9-4d8d-8542-2f6c37a7f5d3">

**Comment by abose** [comment]
Those are not server settings, phcode is a static website and does not have any server. 

Phcode creates a virtual file system in your browser. The option you are seeing are some phcode related virtual system files completely local in your mac. It lives in your browser and does not leave your mac. The diagnostic tools are meant for debugging phcode itself.

**Reply by abose** [reply]
Closing this as its a local diagnostic tool item and not a backend security issue or server leak as the issue suggested.
